firebase-firestore/src/main/java/com/google/firebase/firestore/DocumentChange.java

```diff
@@ -110,26 +111,26 @@ public QueryDocumentSnapshot getDocument() {
   }
 
   /**
-   * The index of the changed document in the result set immediately prior to this DocumentChange
-   * (i.e. supposing that all prior DocumentChange objects have been applied). Returns -1 for
-   * 'added' events.
+   * The index of the changed document in the result set immediately prior to this {@code
+   * DocumentChange} (assuming that all prior {@code DocumentChange} objects have been applied).
+   * Returns -1 for 'added' events.
    */
   @PublicApi
   public int getOldIndex() {
     return oldIndex;
   }
 
   /**
-   * The index of the changed document in the result set immediately after this DocumentChange (i.e.
-   * supposing that all prior DocumentChange objects and the current DocumentChange object have been
-   * applied). Returns -1 for 'removed' events.
+   * The index of the changed document in the result set immediately after this {@code
+   * DocumentChange} (assuming that all prior {@code DocumentChange} objects and the current {@code
+   * DocumentChange} object have been applied). Returns -1 for 'removed' events.
    */
   @PublicApi
   public int getNewIndex() {
     return newIndex;
   }
 
-  /** Creates the list of DocumentChanges from a ViewSnapshot. */
+  /** Creates the list of {@code DocumentChange}s from a {@code ViewSnapshot}. */
```

Comment:
pluralizing with a "s" outside the `{@code block}` ends up rendering with mixed fonts in a single word and is ugly.

Consider rephrasing to "list of `{@code DocumentChange}` instances" (or "objects") or "list of document changes".

Reply:
Used 'document changes' (since I have to address this one individually.)